src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1486,6 +1486,7 @@ internal enum ErrorCode
         ERR_DefaultLiteralNotValid = 8312,
         WRN_DefaultInSwitch = 8313,
         ERR_PatternWrongGenericTypeInVersion = 8314,
+        WRN_UnreferencedLocalFunction = 8315,
```

Comment:
After the review is done, let's rebase to latest dev15.6 branch and move this to the newly added section for C# 7.2.